### PR TITLE
Relocated distributor logos 

### LINKED ISSUE #570: "distributor-logo" icons are spread over two directories
As the title says, `distributor-logo-*.svg` icons are currently placed in two directories, in `apps/scalable` and in `places/scalable`; some are duplicates, some are present in only one dir.

I don't know if this is wanted or if it is a problem, but considering that the plain `distributor-logo.svg` symlink is present on `places/scalable`, it seems that this is the proper place.

Is this something to fix?

### PR DESCRIPTION
**Pull request by stefanobartoletti**
Moved distributor logos from from `apps` to `places`, fixes #570 
Also cleaned their code a bit in the process.

### CODE REVIEW

File: places/scalable/distributor-logo-archlinux.svg [hunk 1]
```diff
@@ -1,107 +1,15 @@
...
+    <stop offset="0" stop-color="#0a80fd" />
+    <stop offset="1" stop-color="#56c5fa" />
+  </linearGradient>
+  <g stroke-width="1.299897" transform="matrix(1.7142909 0 0 1.7142909 -662.715871 -887.647323)">
+    <circle cx="400.57138" cy="531.79791" fill="url(#b)" r="13.999958" />
+    <circle cx="400.57138" cy="531.79791" fill="url(#a)" r="0" />
+  </g>
+  <path d="m673.99096 786.06661c-1.20204 2.94701-1.92701 4.87468-3.26527 7.73381.82049.86979 1.82767 1.88264 3.46336 3.02662-1.75849-.72364-2.95796-1.4501-3.85435-2.20396-1.71272 3.57396-4.39615 8.66484-9.84162 18.44905 4.27996-2.47084 7.59744-3.99415 10.6896-4.57546-.13276-.57103-.20827-1.18872-.20314-1.83321l.004-.13713c.0679-2.74205 1.4943-4.85066 3.18399-4.70747 1.6897.14313 3.00312 2.48339 2.9352 5.22544-.0127.51602-.071 1.01231-.17264 1.4727 3.05841.5983 6.34064 2.11771 10.56267 4.55513-.83252-1.53265-1.57559-2.91428-2.28517-4.23012-1.11779-.86634-2.28362-1.99383-4.66176-3.21451 1.63461.42475 2.80496.91477 3.71723 1.46252-7.21462-13.43273-7.79862-15.21782-10.27303-21.02409z" fill="#fff" fill-opacity=".85" fill-rule="evenodd" transform="matrix(1.03209074 0 0 1.03209074 -671.404402 -802.591814)" />
```
Comment (keeferrourke) [inline places/scalable/distributor-logo-archlinux.svg:14]: Am I crazy, or has the icon size been increased here? The rich diff seems to show a lack of padding between the image border and the icon.

... If I'm not crazy, please resize this icon back as it was before.

Comment (keeferrourke) [reply places/scalable/distributor-logo-archlinux.svg:14]: Okay I'm not crazy! This also happened for the Debian, Fedora, Gentoo, Mageia, Manjaro, OpenSUSE, Rosa, and Ubuntu distributor logos.

**Review by keeferrourke — CHANGES_REQUESTED** [review]
LGTM, except for the icons I pointed out that seem to have been randomly resized.

Also note that `distributor-logo.svg` was a symlink to `distributor-logo-gnome.svg` prior to this change-set. In this PR, it's a copy of the Ubuntu distributor logo. I bet your local configuration overwrote this, so can you please revert that file?

**Comment by stefanobartoletti** [comment]
Sorry for the mess, the script that I use to clean svg files has probably taken way too much files.
I'll fix this.

**Comment by stefanobartoletti** [comment]
Ok, I did a quick research, some icons were present both on `apps` and on `places`, so when I move them these were overwritten. 
Anyway, the difference on the size of many of these distributor logos icons is something already present in the master branch, (maybe because these were done in different times by different persons?).

As an example, the Arch Linux icon is present in both directories, with the icon itself being 48px in one and 44px in another (the canvas size is 48px for both). 

I'm pretty sure that there are also various other circular app icons that are 46px wide.

I'm not really sure which size is supposed to be the standard. What's the correct way here?

**Comment by keeferrourke** [comment]
I think we want to standardize on a 44px icon centered on a 48px canvas.

**Comment by stefanobartoletti** [comment]
I've resized all these icons to be 44x44 inside a 48x48 canvas.

Now they should all adhere to the standard.

Please let me know if this fixes the issue.

**Review by keeferrourke — APPROVED** [review]
LGTM!

**Pull request merged**